### PR TITLE
[docs] Add README newline at EOF

### LINKED ISSUE #884: [gen-helm-docs.sh] Fix End of Files pre-commit fail
# Details

**Helm chart name and version:** All

**Container name and tag:** N/A

**What steps did you take and what happened:**

- Update chart.
- Run `task chart:helm-docs CHART=chartname CHART_TYPE=stable` to update `README.md`
- Push pr.
- pr fails `Pre-commit consistency check` at `Fix End of Files` for `README.md`.

Or

- Update chart.
- Run `task chart:helm-docs CHART=chartname CHART_TYPE=stable` to update `README.md`
- Push pr.
- `pre-commit` fails at `Fix End of Files` for `README.md`.

```shell
$ git commit -m "my message"
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing charts/stable/chartname/README.md
```

**Relevant Helm values:** N/A

**What did you expect to happen:**

- Expect to pass the `Fix End of Files` `pre-commit` after generating the `README.md` using `gen-helm-docs.sh` or `Task`.

**Anything else you would like to add:**

- Re-adding the `README.md` after it's fixed from `pre-commit` seems to work but it would be nicer if this was fixed automatically. It also requires `pre-commit` hooks to be installed to prevent failure on pr submittal, which might be what we want anyway.

- We might be able to modify  `gen-helm-docs.sh` to fix the end of the file.

- Another option is to run `pre-commit` before committing but the user needs to know to do this.

**Additional Information:**

### PR DESCRIPTION
**Pull request by bjw-s**
**Description of the change**

Modified the README template to contain a newline at EOF.

**Benefits**

Pre-commit checks will no longer fail on this

**Possible drawbacks**

None

**Applicable issues**

- fixes #884 

**Additional information**

### CODE REVIEW

**Comment by bjw-s** [comment]
The chart-related checks won't run on this PR, so I'll force merge this after approval :)

**Review by onedr0p — APPROVED** [review]

**Pull request merged**
